src/main/scala/it/pagopa/interop/agreementprocess/api/impl/AgreementApiServiceImpl.scala

```diff
@@ -643,6 +654,7 @@ final case class AgreementApiServiceImpl(
 
     for {
       uid    <- getUidFutureUUID(contexts)
+      _      <- if (agreement.state == Draft && newState == Pending) validateConsumerEmails(agreement) else Future.unit
```

Comment:
Agreement submission can also directly pass from Draft to Active
```suggestion
      _      <- if (agreement.state == Draft) validateConsumerEmails(agreement) else Future.unit
```